web-marketplace/src/lib/queries/react-query/registry-server/getPaymentMethodsQuery/getPaymentMethodsQuery.constants.ts

```diff
@@ -0,0 +1 @@
+export const GET_PAYMENT_METHODS_QUERY_KEY = 'getPost';
```

Comment:
oops forgot to update that in my PR, updated there: https://github.com/regen-network/regen-web/pull/2477/commits/ad280a358c6b2482922816dbb8ea07eea95b9de2
you should do the same here